src/cgame/cg_buildable.cpp

```diff
@@ -1156,8 +1157,19 @@ void CG_GhostBuildable( int buildableInfo )
 
 	ent.hModel = cg_buildables[ buildable ].models[ 0 ];
 
-	ent.customShader = ( SB_BUILDABLE_TO_IBE( buildableInfo ) == IBE_NONE )
-	                     ? cgs.media.greenBuildShader : cgs.media.redBuildShader;
+	int mask = SB_BUILDABLE_TO_IBE( buildableInfo );
```

Comment:
Why is it called `mask`? It's not like a set of bit flags

Reply:
I'm not saying I understand what IBE thingy is, but I probably grepped and found this line: ` #define SB_BUILDABLE_TO_IBE(x)   ( ( ( x ) & SB_BUILDABLE_STATE_MASK ) >> SB_BUILDABLE_STATE_SHIFT )` which likely made me think it really is a mask (yes: probably and likely, I don't remember, I worked on other stuff since then).